### PR TITLE
Use a proper definition list for the account settings page

### PR DESCRIPTION
**Pull request by JammingBen**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15055/

<p>



















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15056/

<p>



















</p></details>

File: packages/web-runtime/src/pages/account.vue [hunk 1]
```diff
@@ -18,33 +18,55 @@
       </div>
       <hr />
       <div class="oc-text-bold oc-mb"><translate>Account Information</translate></div>
```
Comment (pascalwengerter) [inline packages/web-runtime/src/pages/account.vue:20]: Could I interest you in changing this one into a `<h2>` tag for bonus accessibility? :)

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
Almost good to go, me gusta mucho!

File: packages/web-runtime/src/pages/account.vue (hunk 1 above)
Comment (kulmann) [reply packages/web-runtime/src/pages/account.vue:20]: +1 for that 😁

**Review by pascalwengerter — APPROVED** [review]
Great job 🙌🏽

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15098/

<p>










</p></details>

**Pull request merged**